README.md

```diff
@@ -10,37 +10,159 @@ This is a port of [git-machete](https://github.com/VirtusLab/git-machete#git-mac
+
+![](docs/slide_out2.gif)
+
+### Slide in branch
+
+You can also create new branch bellow selected one (also without manual editing `machete` file). To do this, choose `Slide In Branch Bellow Selected Branch` action from context menu.
+Dialog window will appear.
+In text box type new branch name and click `SLide In` button.
+Another dialog will appear - it is a standard new branch IntelliJ dialog.
+Here you can choose if you want to check out a newly created branch or not.
+
+![](docs/slide_in1.gif)
+
+This action can also be used to reattach existing branch bellow selected.
+In this case you just must type name of existing branch that you want to reattach and then click `Slide In`.
+You can also select `Reattach children` checkbox to move all children of reattached branch with it.
+
+![](docs/slide_in2.gif)
+
+![](docs/slide_in3.gif)
+
+### Other actions
+
+On the left side bar you can find another actions (from top to bottom):
+- ![](docs/left_bar_actions/refresh.png) **Refresh Status** - refreshing (reload) branch tree (the graph displayed in main plugin window)
+- ![](docs/left_bar_actions/toggle_listing_commits.png) **Toggle Listing Commits** - show or hide commits belonging to branches (for more details see section [Listing commits](#listing-commits))
+- ![](docs/left_bar_actions/open_machete_file.png) **Open Machete File** - open `machete` file in IntelliJ editor (to see what this editor can do see [Machete file editor](#machete-file-editor) section)
+- ![](docs/left_bar_actions/fetch_all_remotes.png) **Fetch All Remotes** - equivalent of `git fetch --all` command
+- ![](docs/left_bar_actions/push_current_branch.png) **Push Current Branch** - push branch that is currently checked out (shortcut of [Push](#push) context menu action for the current branch)
+- ![](docs/left_bar_actions/rebase_current_branch.png) **Rebase Current Branch Onto Parent** - rebase currently checked out branch onto its parent (shortcut of [Rebase](#rebase) context menu action for the current branch)
+- ![](docs/left_bar_actions/help.png) **Show Help Window** - show window with example branch tree and explanation what parts of this graph mean
+
+### Multi-repository support
+
+Git Machete support many git repositories in one project. It can be many standard repositories or subrepositories.
+If more than one repository will be detected, then selection list will appear in top of main plugin window.
+In that case each repository will have its own `machete` file.
+
+![](docs/multi_repo.gif)
+
+
+## Installing
+
+### Install from JetBrains Marketplace
+
+This plugin is available on [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/14221-git-machete). <br/>
+To install this plugin go to `File` -> `Settings` -> `Plugins`, then make sure you are on `Marketplace` tab (not `Installed`), in search box type `Git Machete` and click `Install`.
+Plugin will be installed. After installation click `Restart IDE` and confirm that action in a messagebox.
```

Comment:
Nit: as for now it applies to 2020.1 but AFAIK not 2020.2 since for some reason the plugin can be updated in 2020.2 without IDE restart (although this is subject to change as the SDK features used in our plugin.xml change); so pls mention that the *restart might be needed depending on IDE version* or sth like that